src/cmd/factory/impl/common.go

```diff
@@ -224,7 +224,8 @@ var (
 
 func generateAndRestoreDriveItems(
 	ctrl *m365.Controller,
-	resourceOwner, secondaryUserID, secondaryUserName string,
+	protectedResource idname.Provider,
+	secondaryUserID, secondaryUserName string,
```

Comment:
would it be possible to remove one of these? It seems like we're passing in the same value but cased differently

Reply:
It's technically usable for onedrive, in case we want the difference, so I'm going to keep it in for now.  We can revisit as needed.